docs/oauth.md

```diff
@@ -69,42 +109,49 @@ You still need to define other required attributes for your connector, `authenti
 
 The OAuth Config file defines your connector's OAuth configuration and also provides the ability to customize how the OAuth flow should work.
 
-The OAuth Config file ([XSD](https://github.com/tableau/connector-plugin-sdk/blob/master/validation/oauth_config.xsd)) is identified in the manifest using the `<oauth-config>` element. Here we discuss the structure of this file.
+The OAuth Config file ([XSD](https://github.com/tableau/connector-plugin-sdk/blob/master/validation/oauth_config.xsd)) is identified in the main plugin manifest using the `<oauth-config>` element. Here we discuss the structure of this file.
 
 ## XML Elements
 
-### `<pluginOAuthConfig>`
-
-This is the parent element for all fields.
+`<pluginOAuthConfig>`
+  is the parent element for all fields below.
 
-### `Elements`
 
-Each OAuth config attribute is represented by an element in the XML, the element name is the attribute name and the content is the attribute value.
 
 | Name  | Type | Meaning | Required? | Notes |
 | ----  | ------- | --------- | ----------- | ----------- |
-| dbclass | String | The identifier for your oauthConfig | Yes | The dbclass must be same with as the `class` attribute in manifest.xml |
+| dbclass | String | The connector class which this OAuth config applies to. | Yes | The dbclass must be same with as the `class` attribute in manifest.xml |
+| oauthConfigId | String | Unique ID for this OAuth config | No |  New in Tableau 2023.1. This is a required attribute if there are multiple OAuth configs defined for a connector. **When using an external config this must begin with the prefix "custom_".**
```

Comment:
When using an external/custom oauth config, this must begin with the prefix "custom_"